apps/gallery/test/marionette/empty_image_test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var Gallery = require('./lib/gallery.js'),
+    assert = require('assert');
+
+marionette('run gallery without any files', function() {
+
+  var app, client;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true,
+        'webgl.force-enabled': true
```

Comment:
Please check and include settings that are must have, I think webgl.force-enabled is not needed for this test